src/raiden_libs/cli.py

```diff
@@ -133,23 +116,42 @@ def blockchain_options(contracts_version: str = None) -> Callable:
         ),
     ]
 
+    arg_for_contract = {
+        CONTRACT_TOKEN_NETWORK_REGISTRY: 'registry',
```

Comment:
Maybe we should make the name more specific: `token-network-registry` and use the `contract`-suffix as well

Reply:
I didn't want to break compatibility in this PR. We would not only need doc changes, but also break existing scenarios. Do we still specify contract addresses in the scenarios we use, or do we already use the default addresses of the official deployments?

Reply:
I'm not sure, but I'd say better do it now than later.

Reply:
But we can also do this in a separate PR.